price-feeder/oracle/provider/okx.go

```diff
@@ -175,6 +217,22 @@ func (p *OkxProvider) setTickerPair(tickerPair OkxTickerPair) {
 	p.tickers[tickerPair.InstId] = tickerPair
 }
 
+func (p *OkxProvider) setCandlePair(pairData []string, instID string) {
+	p.mu.Lock()
+	defer p.mu.Unlock()
+
+	ts, err := strconv.ParseInt(pairData[0], 10, 64)
+	if err != nil {
+		ts = 0
```

Comment:
do we still want to have this set if we can't parse the time?

Reply:
returned here instead 👍